static/src/stylesheets/module/signup/_newsletters.scss

```diff
@@ -6,6 +6,58 @@
     padding-bottom: 16px;
 }
 
+.newsletters-heading__divider {
+    margin-bottom: 16px;
+    border-bottom: 1px solid $brightness-86;
+}
+
+.newsletters-privacy-notice__container {
+    position: sticky;
+    top: 0;
+    background-color: #ffe500;
```

Comment:
I think we can use `$highlight-main` here

Reply:
updated